solutions/deploy/cloud-pak-deployer/scripts/uninstall.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+${OC} create -f "${JOB_UNINSTALL_CPD_FILENAME}"
+RC=$?
+if [ "${RC}" -ne 0 ]; then echo "Unable to create job ${JOB_NAME}; exiting..." && exit 1; fi
+
+timeout_seconds=1800 # 30 minutes
+sleep_seconds=5
+number_of_tries=$((timeout_seconds / sleep_seconds))
+complete=false
+failed=false
+
+i=0
+while [ $i -lt "${number_of_tries}" ]; do
+
+echo "Running job ... $i"
+${OC} get jobs -n "${NAMESPACE_NAME}"
+
+${OC} wait --for=condition=complete job "${JOB_NAME}" -n "${NAMESPACE_NAME}" --timeout=0 2>/dev/null
```

Comment:
timeout=0 would make it check once (and not wait), so is there a risk of timing issue here?